vendor.conf

```diff
@@ -10,7 +10,7 @@ github.com/containerd/cgroups 77e628511d924b13a77cebdc73b757a47f6d751b
 github.com/containerd/continuity master
 github.com/containernetworking/cni v0.6.0
 github.com/containernetworking/plugins 1fb94a4222eafc6f948eacdca9c9f2158b427e53
-github.com/containers/image ad33f7b73fbac0acf05b9e2cea021b61b4b0c3e0
+github.com/containers/image 5144ced37a1b21b63c6ef605e56811e29a687528
```

Comment:
Is this the same vendor @haircommander is doing?

Reply:
yes ignore mine